package.json

```diff
@@ -6,8 +6,11 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "pages dev",
-    "build": "vite build",
+    "dev": "yext pages build && yext pages generate-test-data && pages dev",
+    "build:local": "vite build",
+    "build": "yext pages build && yext pages generate-test-data && yext pages render",
+    "build:serve": "yext pages build && yext pages generate-test-data && yext pages render && yext pages serve",
```

Comment:
I'd just have this call `npm run build && npm run serve` to keep it simpler. Also, if you change one command in the future you don't need to change in both places.

Reply:
@mkilpatrick done.